source/operations/monitoring/metrics-and-alerts.rst

```diff
@@ -15,80 +15,87 @@ Metrics and Alerts
+.. code-block:: shell
+   :class: copyable
 
-      .. code-block:: shell
-         :class: copyable
+   http://HOSTNAME:PORT/minio/metrics/v3/audit
 
-         http://HOSTNAME:PORT/minio/v2/metrics/cluster
+Replace ``HOSTNAME:PORT`` with the :abbr:`FQDN (Fully Qualified Domain Name)` and port of the MinIO deployment.
+For deployments with a load balancer managing connections between MinIO nodes, specify the address of the load balancer.
 
-      Replace ``HOSTNAME:PORT`` with the :abbr:`FQDN (Fully Qualified Domain Name)` and port of the MinIO deployment.
-      For deployments with a load balancer managing connections between MinIO nodes, specify the address of the load balancer.
+By default, MinIO requires authentication to scrape the metrics endpoints.
+To generate the needed bearer tokens, use :mc:`mc admin prometheus generate`.
+You can also disable metrics endpoint authentication by setting :envvar:`MINIO_PROMETHEUS_AUTH_TYPE` to ``public``.
 
+MinIO provides the following scraping endpoints, relative to the base URL:
 
-   .. tab-item:: Bucket Metrics
-
-      .. versionchanged:: RELEASE.2023-08-31T15-31-16Z
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+   :width: 100%
 
-      You can scrape :ref:`bucket-level metrics <minio-available-bucket-metrics>` using the following URL endpoint:
-
-      .. code-block:: shell
-         :class: copyable
-
-         http://HOSTNAME:PORT/minio/v2/metrics/bucket
-
-      Replace ``HOSTNAME:PORT`` with the :abbr:`FQDN (Fully Qualified Domain Name)` and port of the MinIO deployment.
-      For deployments with a load balancer managing connections between MinIO nodes, specify the address of the load balancer.
-
-   .. tab-item:: Resource Metrics
-
-      .. versionadded:: RELEASE.2023-10-07T15-07-38Z 
-
-      You can scrape :ref:`resource metrics <minio-available-resource-metrics>` using the following URL endpoint:
-
-      .. code-block:: shell
-         :class: copyable
+   * - Category
+     - Path
 
-         http://HOSTNAME:PORT/minio/v2/metrics/resource
+   * - API
+     - ``/api/requests``
+       
+       ``/bucket/api``
 
-      Replace ``HOSTNAME:PORT`` with the :abbr:`FQDN (Fully Qualified Domain Name)` and port of the MinIO deployment.
-      For deployments with a load balancer managing connections between MinIO nodes, specify the address of the load balancer.
+   * - Audit
+     - ``/audit``
 
+   * - Cluster
+     - ``/cluster/config``
+       
+       ``/cluster/erasure-set``
+       
+       ``/cluster/health``
+       
+       ``/cluster/iam``
+       
+       ``/cluster/usage/buckets``
+       
+       ``/cluster/usage/objects``
 
-MinIO by default requires authentication for scraping the metrics endpoints.
-Use the :mc:`mc admin prometheus generate` command to generate the necessary bearer tokens. 
-You can alternatively disable metrics endpoint authentication by setting :envvar:`MINIO_PROMETHEUS_AUTH_TYPE` to ``public``.
+   * - Debug
```

Comment:
The metrics list source, `v3.md`, doesn't list individual metrics for the Debug section. They come from Go, and there isn't a good list that I can find. Not sure the best thing to say about that.

Reply:
[Prometheus goLang client context](https://github.com/prometheus/client_golang/blob/46f77a97fa1e83e7285634745bd5c92c11bf20c7/prometheus/collectors/go_collector_go116.go#L21) and [Prometheus goLang client go.Collector](https://github.com/prometheus/client_golang/blob/46f77a97fa1e83e7285634745bd5c92c11bf20c7/prometheus/go_collector.go#L209) seem like the likely places.

Reply:
I don't know that we'd want to recreate that list or try to link to a GitHub page. Maybe:

```suggestion
* - Debug
    The debug endpoint uses standard metrics from the :ref:`Prometheus Go Client repository's <https://github.com/prometheus/client_golang>`__ base collector.
```

Reply:
Whatever it is this points to, I think the link should be in the original md file. Will make a PR for that in `minio/minio` and see if I can get some additional feedback from eng.

Reply:
Proposed update in the original looks like this:

![Revised text in list of metrics Debug section: Standard Go runtime metrics from the Prometheus Go Client base collector.](https://github.com/user-attachments/assets/7e1e7787-ae23-4a9b-87eb-c1b1b80e3b33)

Reply:
The `v3.md` change merged, staging updated